docs/release-notes/may-2024.md

```diff
@@ -0,0 +1,141 @@
+
+### Changed
+
+- The sorting criteria used on Semgrep AppSec Platform's **Findings** has been updated to reflect the follow order:
+  1. Severity
+  2. Findings generated by custom rules
+  3. Findings generated by [Pro rules](/semgrep-code/pro-rules)
+  4. Issue count in descending order
+  5. Findings ID in ascending order
+
+### Fixed
+
+- When using `semgrep --test --json` to run tests against your rules and obtain the results in JSON format, Semgrep now reports the following issues to the `config_missing_fixtests` field in the JSON output for all rules in a file (not just the first rule):
+  - Rule files containing `fix:` without the corresponding `.fixed` test file.
+  - Rule files using `fix-regex:` without the corresponding `.fixed` test file.
+- Fixed an issue where Dockerfiles lacking a trailing newline character at the end of the file caused a segmentation fault.
+- Fixed an issue with the improper handling of Unicode characters caused Semgrep to crash.
+- Fixed an issue where interfile tainting missed a constant propagation phase, leading to the omission of true positives in some cases during interfile analysis.
+- Fixed an issue where Semgrep ignored YAML tags instead of matching them correctly.
+- Fixed an issue where findings identified in an earlier scan aren't marked as fixed when they no longer appear in later scans.
+- Fixed an issue where patterns flagged as disabled were not disabled when switching from structure mode to advanced mode in the Semgrep Editor.
+- **CLI**:
+  - When outputting Semgrep results in SARIF format, Semgrep now adds the **security** tag when CWE metadata is present in the rule.
+  - Fixed an issue where rules with `metavariable-type` do not show up in the SARIF output.
+
+## ⛓️  Semgrep Supply Chain
+
+### Added
+
+- Added public Supply Chain APIs. [<i class="fas fa-external-link fa-xs"></i> Read the documentation](https://semgrep.dev/api/v1/docs/#tag/Finding/operation/semgrep_app.core_exp.findings.handlers.issue.openapi_list_recent_issues).
+- Added [lockfile-only support](/supported-languages/#semgrep-supply-chain) for the **Hex package manager** for Elixir codebases.
+- Added two new triage statuses: **Reviewing**, which indicates the findings that require more investigation to determine what the next steps should be, and **Fixing**, which indicate findings for which a Jira ticket has been created.
+
+### Changed
+
+- The Supply Chain UI has been improved for consistency across Semgrep products. With these changes, users are now able to easily manage SCA, SAST, and secret findings. The Supply Chain UI updates provide the following workflow improvements:
+  - Grouping vulnerabilities by rule
+  - Bulk triaging of findings
+  - Comprehensive filtering
+  - A unified API for findings across Semgrep Code and Semgrep Supply Chain
+- License policy
```

Comment:
@s-santillan I think there should be more here?

Reply:
Oh that's wrt for Ruby license policies. It may be an implementation detail, I'll remove it.